deps/homalg-project.jl

```diff
@@ -281,7 +281,7 @@ called once `GAP.jl` gets updated.
 """
 global PACKAGES_TO_COMPILE = [
     "Gauss",
-    #"Browse", ## do not compile browse as it results into GAP raising the error "Error opening terminal: xterm-256color."
+    "Browse",
```

Comment:
By chance I run into this error today, and then came upon this PR (for unrelated reasons) 😂 

Anyway, the last comment on https://github.com/oscar-system/GAP.jl/issues/456 may be of interest to you

Reply:
Thank you for the hint. I will follow the discussions there :)